### PR TITLE
gcta: add new version

### PR DESCRIPTION
**Pull request by noguchi-k**
Added a new version `1.93.1` because we could not find the URL that can get version `1.91.2`.

### CODE REVIEW

**Comment by adamjstewart** [comment]
We usually keep the old version in the package in case someone needs to reproduce a build and the tarball is still in their cache. See #15896 for more discussion.

**Comment by noguchi-k** [comment]
I added the old version.

File: var/spack/repos/builtin/packages/gcta/package.py [hunk 1]
```diff
@@ -15,11 +15,15 @@ class Gcta(Package):
...
     homepage = "http://cnsgenomics.com/software/gcta/#Overview"
-    url      = "http://cnsgenomics.com/software/gcta/gcta_1.91.2beta.zip"
+    url      = "https://cnsgenomics.com/software/gcta/bin/gcta_1.93.1beta.zip"
 
-    version('1.91.2beta_mac', 'ce0882ad35dd9474ffe40911da369274700af1ecb9916c0a355b7bad14850234')
-    version('1.91.2beta', '192efb767be1c7ca9c2dac5d2c2317a97c7a9db1f801168d19ad2a51b98d9b10', preferred=True)
+    version('1.93.1beta_mac', '86fb7f1885beaa3e35d39a89c9ac9522d5b337bffcfc2194fc2f886d13cda823')
+    version('1.93.1beta', 'e6439fc0173642d917a039dbdc6a8cb5b309f76d4f56762212b1a760a2c8a678', preferred=True)
+    version('1.91.2beta_mac', 'ce0882ad35dd9474ffe40911da369274700af1ecb9916c0a355b7bad14850234', deprecated=True)
+    version('1.91.2beta', '192efb767be1c7ca9c2dac5d2c2317a97c7a9db1f801168d19ad2a51b98d9b10', deprecated=True)
```
Comment (alalazo) [inline var/spack/repos/builtin/packages/gcta/package.py:23]: As far as I know the `deprecated` keyword is ignored (i.e. we don't have a feature to deprecate versions based on it), or am I missing something?

Comment (adamjstewart) [reply var/spack/repos/builtin/packages/gcta/package.py:23]: Correct, until #15896 is addressed this keyword will be ignored. I would remove it until a consensus on #15896 has been reached.

Comment (adamjstewart) [reply var/spack/repos/builtin/packages/gcta/package.py:23]: Ping @noguchi-k, can you remove `deprecated=True` from these?

**Comment by alalazo** [comment]
We have recently added the possibility to annotate a version as deprecated. Can you rebase and update the PR?

**Comment by alalazo** [comment]
Closing as stale

**Pull request closed without merging**